holoviews/plotting/renderer.py

```diff
@@ -271,7 +271,7 @@ def get_plot_state(self_or_cls, obj, renderer=None, **kwargs):
         Given a HoloViews Viewable return a corresponding plot state.
         """
         if not isinstance(obj, Plot):
-            obj = self_or_cls.get_plot(obj, renderer, **kwargs)
+            obj = self_or_cls.get_plot(obj=obj, renderer=renderer, **kwargs)
```

Comment:
For `BokehRenderer.get_plot` plot the signature is: 
``` python
@bothmethod
def get_plot(self_or_cls, obj, doc=None, renderer=None, **kwargs):
```
So a (small) bug as I see it.